src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+type Props = {
+  title: string;
+};
+
+export const TodoItem: React.FC<Props> = ({ title }) => {
+  return (
+    <div data-cy="Todo" className="todo">
+      <label className="todo__status-label">
```

Comment:
You can read more here for fix this error 
https://stackoverflow.com/questions/54446655/eslint-rule-for-label